Installing-Oppia.md

```diff
@@ -17,19 +17,23 @@ following:
 
 If you need to learn or brush up on these, please do so before trying to start the installation so you don't get confused by our instructions.
 
-## Standard installation using Python setup
+## Python setup (soon to be depreciated)
```

Comment:
Thanks @jayam04 -- can we put this section below the Docker one please?

Reply:
done!